docker-compose.override.yml

```diff
@@ -0,0 +1,5 @@
+version: "2.3"
```

Comment:
It would be nice to see a comment at the top explaining that this is used for functional tests